tools/query_sloop.c

```diff
@@ -0,0 +1,22 @@
+#include <acme.h>
+static inherit AcmeShadow;
+#include AcmeShadowInc
```

Comment:
I prefer some whitespace between inherits and includes. Here's it's not as big deal because you've only got one of each, but usually you'll have a block of stuff and you want those blocks to be distinct.